cmd/gapit/export_replay.go

```diff
@@ -98,5 +114,155 @@ func (verb *exportReplayVerb) Run(ctx context.Context, flags flag.FlagSet) error
 	if err := client.ExportReplay(ctx, capturePath, device, verb.Out, opts); err != nil {
 		return log.Err(ctx, err, "Failed to export replay")
 	}
+
+	if verb.Apk != "" {
+		// Create stand-alone APK
+		replayAPK := filepath.Base(verb.Apk)
+		replayPackage := strings.TrimSuffix(replayAPK, ".apk")
+		log.I(ctx, "Create replay apk: %s with package name %s", replayAPK, replayPackage)
+
+		boxedCapture, err := client.Get(ctx, capturePath.Path(), nil)
+		if err != nil {
+			return log.Err(ctx, err, "Failed to load the capture")
+		}
+		capture := boxedCapture.(*service.Capture)
+
+		refAPKPath, err := layout.GapidApk(ctx, capture.ABI)
+		if err != nil {
+			return err
+		}
+
+		// Extract reference APK
+		if err := execCommand(ctx, "unzip", "-o", refAPKPath.String()); err != nil {
+			return err
+		}
+
+		// Create new manifest
+		f, err := os.Create("NewManifest.xml")
+		if err != nil {
+			return err
+		}
+		t, err := template.New("manifest").Parse(manifest())
+		if err != nil {
+			return err
+		}
+		type ManifestEntries struct {
+			Package string
+		}
+		var entries = ManifestEntries{replayPackage}
+		t.Execute(f, entries)
+		f.Close()
+
+		// Re-assemble new APK with the new manifest
+		if err := execCommand(ctx, "aapt2", "link", "-o", verb.Apk, "--manifest", "NewManifest.xml", "-I", os.ExpandEnv("${ANDROID_SDK_HOME}/platforms/android-26/android.jar")); err != nil {
```

Comment:
We should use aapt2 from the ANDROID_SDK_HOME.
Also it would be good to be able to specify the android_sdk_home from the command-line as well.

`--sdk_root="foo"`.  If not set then use the env var.